src/sites/doc/docs/taro/theme-react.en-US.md

```
@@ -0,0 +1,58 @@
+# Customize Theme
+
+## Introduce
+
+NutUI-React supports flexible style customization to meet multiple visual business and brand needs, including but not limited to support for global master tone and component-specific visual customization.
+<br />
+<br />
+In NutUI-React version 1.4.0, the Sass theme customization function of 1.x can still be used. We added CSS variables to the original theme customization function, so that the new theme customization function can be used without introducing additional SCSS style files.
+
+## Method 1: Use CSS Variables to configure topics
+
+`NutUI - React` can [CSS Vars] (https://developer.mozilla.org/zh-CN/docs/Web/CSS/Using_CSS_custom_properties) to organize the pattern, By overwriting these 'CSS' variables, you can achieve custom theme, dynamic switching theme and other functions.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)